ansible/roles/wifi-ap/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 ssid: "ConnectBox - Free Media"
+wireless_channel: 3
```

Comment:
Alright, I also decided it also made more sense to put this back at channel 1 and override the channel in the orange_pi_zero group vars instead (with comments)